compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -502,6 +502,18 @@ static bool canUseInOperandAsInitOperand(OpOperand *inOperand,
   return true;
 }
 
+/// All operations in a dispatch should be vectorized, which isnt the case today
+/// This is an explicit list of operations that arent vectorized for now
+/// requiring special handling for now in dispatch region formation to avoid
+/// large stack allocations.
+static bool isVectorizedAlways(Operation *producer) {
+  if (auto convOp = dyn_cast<linalg::Conv2DNhwcHwcfOp>(producer)) {
+    auto strides = convOp.getStrides();
+    return strides.isSplat() && strides.getSplatValue<int64_t>() == 1;
```

Comment:
nit: Check dilations here too

Reply:
I am walking this slowly a bit. This is under the "aggressive-fusion" flag that we start using with SDXL and more things here as we go. Eventually this method should just return true always. So No point generalizing this.